src/examples/component-definition/xml/example-component-definition.xml

```diff
@@ -46,17 +49,52 @@
         source="https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_MODERATE-baseline_profile.xml">
       <description>
         <p>MongoDB control implementations for NIST SP 800-53 revision 5.</p>
-      </description>
+      </description>          
       <implemented-requirement
-          uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee2a0"
-          control-id="sc-8.1">
+          uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee080"
+          control-id="sc-8">
+        <description>
+          <p>MongoDB's implementation of SC-8 control</p>
+        </description>
```

Comment:
This description doesn't add any information. You should consider revising it to containing some additional information. Maybe an explanation that the control implementation is addressed at the statement level and there are no additional details at the control level?

Reply:
Done.